testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2250,7 +2250,10 @@ private List<SetColumnTypeSetup> setColumnTypeSetupData()
     {
         return ImmutableList.<SetColumnTypeSetup>builder()
                 .add(new SetColumnTypeSetup("tinyint", "TINYINT '127'", "smallint"))
+                .add(new SetColumnTypeSetup("tinyint", "TINYINT '127'", "integer"))
```

Comment:
This probably belongs to "Support type evolution from tinyint to smallint and bigint in ORC" (currently in  "Add test cases for changing numeric column types")